src/algorithms/types/JsonToTypes.ts

```diff
@@ -0,0 +1,18 @@
+import { EmpiricalData, EmpiricalDatum } from './Param.types'
+import { CaseCounts } from '../../assets/data/CaseCounts.types'
+
+export function toEmpiricalDatum(input: CaseCounts): EmpiricalDatum {
+  const toNumber = (val: number | null) => (val === null ? 0 : val)
+
+  return {
+    cases: toNumber(input.cases),
+    hospitalized: toNumber(input.hospitalized),
+    deaths: toNumber(input.deaths),
+    ICU: toNumber(input.ICU),
+    time: new Date(input.time),
+  }
+}
```

Comment:
@abrie If we are going for runtime validation here, I would prefer using a library wherever possible. Just to make sure this `toNumber()` don't grow hairs in the form of `toPositibeNumber()` and friends, to avoid writing the entire validation library from scratch.

Additionally, I would prefer declarative validation solutions. We currently use `yup` for Form validaton:
https://github.com/neherlab/covid19_scenarios/blob/73d3b94a578e39e2ea9ee01c5448a3e9a2bf0874/src/components/Main/validation/schema.ts#L12-L54
 and URL param folks are going to add `io-ts` (https://github.com/neherlab/covid19_scenarios/pull/53.)

This is probably an overkill for the `toEmpiricalDatum()` right now, but as we going forward, we need to validate all the external data sources (Form, LocalStorage https://github.com/neherlab/covid19_scenarios/pull/214, URL params https://github.com/neherlab/covid19_scenarios/pull/53, file import https://github.com/neherlab/covid19_scenarios/issues/11, more data files, network, etc.) in a cohesive manner, and to share validation schemas as much as possible to avoid updating multiple sources of truth.